### ISSUE TITLE
pod install 报错

### ISSUE THREAD

**Issue by hreochen** [qiuxiang/react-native-amap-geolocation#154]
使用了这两个库 pod install 会报错

```
 "react-native-amap-geolocation": "^1.2.0",
  "react-native-amap3d": "^3.1.3",

```

[!] CocoaPods could not find compatible versions for pod "AMapFoundation":
  In snapshot (Podfile.lock):
    AMapFoundation (= 1.6.9, ~> 1.6.9)

  In Podfile:
    react-native-amap-geolocation (from `../node_modules/react-native-amap-geolocation`) was resolved to 1.2.0, which depends on
      AMapLocation (~> 2.9.0) was resolved to 2.9.0, which depends on
        AMapFoundation (>= 1.7.0)

**Comment by qiuxiang** [comment]
得更新下 amap3d 这个库的依赖版本

**Comment by hreochen** [comment]
> 得更新下 amap3d 这个库的依赖版本

大佬具体怎么改

**Comment by qiuxiang** [comment]
如果你要临时解决的话，改 `node_modules/react-native-amap3d/react-native-amap3d.podspec`

https://github.com/qiuxiang/react-native-amap3d/blob/8c4e7cca0cdf52836d62f69ff23e4318ecef63aa/react-native-amap3d.podspec#L19

**Comment by qiuxiang** [comment]
本来我要发个新版本的，但 android 那边冲突了，暂时没能解决，先不发了

**Comment by hreochen** [comment]
可以了

**Comment by happystudy666** [comment]
怎么解决的。。。是改成9.4.0就好了吗。。为什么我改了不行。。。

**Comment by Huang-zekai** [comment]
[!] CocoaPods could not find compatible versions for pod "AMapFoundation":
  In snapshot (Podfile.lock):
    AMapFoundation (= 1.6.9, ~> 1.6.9)

  In Podfile:
    react-native-amap-geolocation (from `../node_modules/react-native-amap-geolocation`) was resolved to 1.2.2, which depends on
      AMapLocation (~> 2.9.0) was resolved to 2.9.0, which depends on
        AMapFoundation (>= 1.7.0)


You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * changed the constraints of dependency `AMapFoundation` inside your development pod `react-native-amap-geolocation`.
   You should run `pod update AMapFoundation` to apply changes you've made.
![image](https://user-images.githubusercontent.com/82138184/236616925-7ff45b9b-f120-4ae5-adaa-e539db08dde6.png)修改成9.4.0还是不行
